src/components/timeout-dialog/timeout-dialog.js

```diff
@@ -182,20 +185,24 @@ function TimeoutDialog($module, $sessionActivityService) {
       '<button id="hmrc-timeout-keep-signin-btn" class="govuk-button">',
       settings.keepAliveButtonText,
     );
-    const $signOutButton = utils.generateDomElementFromStringAndAppendText(
-      '<a id="hmrc-timeout-sign-out-link" class="govuk-link hmrc-timeout-dialog__link">',
-      settings.signOutButtonText,
-    );
-
-    $staySignedInButton.addEventListener('click', keepAliveAndClose);
-    $signOutButton.addEventListener('click', signOut);
-    $signOutButton.setAttribute('href', settings.signOutUrl);
 
     $element.appendChild($visualMessge);
     $element.appendChild($audibleMessage);
     $element.appendChild($staySignedInButton);
     $element.appendChild(document.createTextNode(' '));
-    $element.appendChild(wrapLink($signOutButton));
+
+    if (!settings.hideSignOutLink) {
+      const $signOutButton = utils.generateDomElementFromStringAndAppendText(
+        '<a id="hmrc-timeout-sign-out-link" class="govuk-link hmrc-timeout-dialog__link">',
```

Comment:
we seem to have a mix of (existing) names - `signOutButton` vs `sign-out-link` which is perhaps confusing for the end user, but maybe not worth fixing now.